app/models/katello/root_repository.rb

```diff
@@ -434,6 +434,23 @@ def format_arches
       end
     end
 
+    def format_custom_content_path
+      path = custom_content_path
+      if content_type == ::Katello::Repository::DEB_TYPE && SETTINGS[:katello][:use_deb_structured_publisher] == true
```

Comment:
Long term story: we want to get rid of this setting again. 
We need this setting so that old sub-man works. If "all users" use the new sub-man (see https://github.com/candlepin/subscription-manager/pull/3223 ) then we would remove this check setting check again. Additionally, we would remove to do a simple publish in pulp which would increase the performance a lot.